src/test/java/net/sf/jabref/importer/fileformat/EndnoteImporterTest.java

```diff
@@ -0,0 +1,126 @@
+package net.sf.jabref.importer.fileformat;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+
+public class EndnoteImporterTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    EndnoteImporter importer;
+
+    @Before
+    public void setUp() throws Exception {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
```

Comment:
Reference in the docs: https://github.com/JabRef/jabref/wiki/Code-Howtos#preferences-in-tests